apps/trading/e2e/fixtures/market.py

```diff
@@ -154,3 +154,88 @@ def setup_continuous_market(vega: VegaService, market_id: str = None, **kwargs):
     vega.wait_for_total_catchup()
 
     return market_id
+
+def setup_perps_market(
+    vega: VegaService,
+    custom_asset_name="tDAI",
+    custom_asset_symbol="tDAI",
+):
+    for wallet in wallets:
+        vega.create_key(wallet.name)
+
+    vega.mint(
+        MM_WALLET.name,
+        asset="VOTE",
+        amount=mint_amount,
+    )
+
+    vega.update_network_parameter(
+        MM_WALLET.name, parameter="market.fee.factors.makerFee", new_value="0.1"
+    )
+    vega.forward("10s")
+    vega.wait_fn(1)
+    vega.wait_for_total_catchup()
+
+    vega.create_asset(
+        MM_WALLET.name,
+        name=custom_asset_name,
+        symbol=custom_asset_symbol,
+        decimals=5,
+        max_faucet_amount=1e10,
+    )
+    vega.wait_fn(1)
+    vega.wait_for_total_catchup()
+    tdai_id = vega.find_asset_id(symbol=custom_asset_symbol)
+    logger.info(f"Created asset: {custom_asset_symbol}")
+
+    vega.mint(
```

Comment:
This _could_ be done in a for loop. List of wallet names ['wallet1', 'wallet2', 'wallet3']
```
for wallet_name in wallet_name_list:
    vega.mint(
        wallet_name,
        asset=tdai_id,
        amount=mint_amount,
    )
```

More scaleable. One of those where if you wanted you could pass in a whole list of wallets as a param too should you wish.

Reply:
Going to leave as is for now, as we will probs refactor this entire file in the coming weeks.